### PR TITLE
[ACA-2365] infodrawer metadata in expanded card

### PR DESCRIPTION
**Pull request by dhrn**
## PR Checklist
Please check if your PR fulfills the following requirements:

```
- [ ] The commit message follows our guidelines: https://github.com/Alfresco/alfresco-content-app/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
```

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application / Infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,10 +36,10 @@
...
   "private": true,
   "dependencies": {
-    "@alfresco/adf-content-services": "3.2.0-beta7",
-    "@alfresco/adf-core": "3.2.0-beta7",
-    "@alfresco/adf-extensions": "3.2.0-beta7",
-    "@alfresco/js-api": "3.2.0-beta7",
+    "@alfresco/adf-content-services": "3.2.0-90276dd4988b80cb0e4b6912b41e8d0c810a313c",
+    "@alfresco/adf-core": "3.2.0-90276dd4988b80cb0e4b6912b41e8d0c810a313c",
+    "@alfresco/adf-extensions": "3.2.0-90276dd4988b80cb0e4b6912b41e8d0c810a313c",
+    "@alfresco/js-api": "~3.2.0-fa5916ff413131513c3e382d7f27dd9b4cfa0e7e",
```
Comment (DenysVuika) [inline package.json]: needs exact version. You can upgrade to the latest alpha with the following command:

```
./scripts/update-version.sh -v alpha
```

**Comment by DenysVuika** [comment]
@dhrn can you please add a couple of unit tests to check the default value (exif) and that it's getting changed via actions?

**Comment by dhrn** [comment]
ok. give me some time. I will update you once it is done

**Comment by dhrn** [comment]
@DenysVuika do we need to update the **diplayAspect** when we close the side nav?

**Comment by DenysVuika** [comment]
@dhrn I think not needed as it's just toggling the visibility

**Comment by dhrn** [comment]
@DenysVuika when will I get this thing on NPM registry?

**Comment by DenysVuika** [comment]
@dhrn There are 2 suites of e2e tests failing right now on Travis

**Comment by dhrn** [comment]
ok. let me have a look

**Comment by dhrn** [comment]
@DenysVuika  what is the command to run e2e locally?

**Comment by DenysVuika** [comment]
@dhrn to test a particular e2e test, use `fit(...)`, then 

1. Running local ACS community:

```
npm run start:docker
```

2. Running local e2e run

```
npm run e2e.local
```

3. Stopping community

```
npm run stop:docker
```

**Comment by dhrn** [comment]
thanks

**Comment by dhrn** [comment]
I seem timeout issue. I am just wondering how this stuff is working for other pr

**Comment by DenysVuika** [comment]
Check other tests maybe. The runs were always green before.

**Comment by dhrn** [comment]
@DenysVuika can you stop my build?

**Comment by dhrn** [comment]
thanks

**Comment by dhrn** [comment]
i am closing this PR. i will create new one

**Pull request closed without merging**